<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="9" passed="9" failed="0" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-06-01T09:53:02 IST" name="Default suite" finished-at="2020-06-01T09:53:02 IST" duration-ms="56">
    <groups>
    </groups>
    <test started-at="2020-06-01T09:53:02 IST" name="Default test" finished-at="2020-06-01T09:53:02 IST" duration-ms="56">
      <class name="TestCalculate">
        <test-method is-config="true" signature="init()[pri:0, instance:TestCalculate@7995092a]" started-at="2020-06-01T09:53:02 IST" name="init" finished-at="2020-06-01T09:53:02 IST" duration-ms="6" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method signature="CustomDelimiter()[pri:0, instance:TestCalculate@7995092a]" started-at="2020-06-01T09:53:02 IST" name="CustomDelimiter" finished-at="2020-06-01T09:53:02 IST" duration-ms="5" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- CustomDelimiter -->
        <test-method signature="EmptyString()[pri:0, instance:TestCalculate@7995092a]" started-at="2020-06-01T09:53:02 IST" name="EmptyString" finished-at="2020-06-01T09:53:02 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- EmptyString -->
        <test-method signature="IgnoreNumbersGreaterThanThousand()[pri:0, instance:TestCalculate@7995092a]" started-at="2020-06-01T09:53:02 IST" name="IgnoreNumbersGreaterThanThousand" finished-at="2020-06-01T09:53:02 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- IgnoreNumbersGreaterThanThousand -->
        <test-method signature="NegativeNumbersThrowexception()[pri:0, instance:TestCalculate@7995092a]" started-at="2020-06-01T09:53:02 IST" name="NegativeNumbersThrowexception" finished-at="2020-06-01T09:53:02 IST" duration-ms="1" status="PASS">
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Negatives are not allowed: -1]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Negatives are not allowed: -1
at Calculator.getAddition(Calculator.java:39)
at Calculator.calculate(Calculator.java:19)
at TestCalculate.NegativeNumbersThrowexception(TestCalculate.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- NegativeNumbersThrowexception -->
        <test-method signature="SingleString()[pri:0, instance:TestCalculate@7995092a]" started-at="2020-06-01T09:53:02 IST" name="SingleString" finished-at="2020-06-01T09:53:02 IST" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SingleString -->
        <test-method signature="TheNegativeNumberReturnInTheExceptionMessage()[pri:0, instance:TestCalculate@7995092a]" started-at="2020-06-01T09:53:02 IST" name="TheNegativeNumberReturnInTheExceptionMessage" finished-at="2020-06-01T09:53:02 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TheNegativeNumberReturnInTheExceptionMessage -->
        <test-method signature="ThreeNumberDelimitedEitherWay()[pri:0, instance:TestCalculate@7995092a]" started-at="2020-06-01T09:53:02 IST" name="ThreeNumberDelimitedEitherWay" finished-at="2020-06-01T09:53:02 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ThreeNumberDelimitedEitherWay -->
        <test-method signature="TwoNumbersDelimitedByNewLineReturnsTheSum()[pri:0, instance:TestCalculate@7995092a]" started-at="2020-06-01T09:53:02 IST" name="TwoNumbersDelimitedByNewLineReturnsTheSum" finished-at="2020-06-01T09:53:02 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TwoNumbersDelimitedByNewLineReturnsTheSum -->
        <test-method signature="TwoStringsDelimitedByCommaReturnsTheSum()[pri:0, instance:TestCalculate@7995092a]" started-at="2020-06-01T09:53:02 IST" name="TwoStringsDelimitedByCommaReturnsTheSum" finished-at="2020-06-01T09:53:02 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TwoStringsDelimitedByCommaReturnsTheSum -->
      </class> <!-- TestCalculate -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
